cmd/list.go

```diff
@@ -39,6 +39,25 @@ var listCmd = &cobra.Command{
 			os.Exit(1)
 		}
 
+		// Filter mods by side
+		if viper.IsSet("list.side") {
+			// cobra/viper ensures this value is set if the flag is used. Maybe remove the len Check?
+			side := viper.GetString("list.side")
+			if side != core.UniversalSide && side != core.ServerSide && side != core.ClientSide {
+				fmt.Printf("Invalid side %q, must be one of client, server, or both (default)\n", side)
+				os.Exit(1)
+			}
+
+			i := 0
+			for _, mod := range mods {
+				if len(side) == 0 || (mod.Side == side || mod.Side == core.EmptySide || mod.Side == core.UniversalSide || side == core.UniversalSide) {
```

Comment:
No need for the len check here - the previous if already validates that the side is one of both/server/client